eventually/authentication/models.py

```diff
@@ -50,6 +66,18 @@ def get_by_id(user_id):
             return None
 
     @staticmethod
+    def get_by_email(user_email):
+        """
+        Returns user by email
+        :param user_email: email by which we need to find the user
+        :return: user object or None if a user with such ID does not exist
+        """
+        try:
+            return CustomUser.objects.get(email=user_email)
+        except CustomUser.DoesNotExist:
+            return None
+
+    @staticmethod
     def delete_by_id(user_id):
         """
         :param user_id: an id of a user to be deleted
```

Comment:
indicate the type of the function parameter with sphinx.

Reply:
Fixed.